dk-installer.py

```diff
@@ -1485,23 +1540,30 @@ def execute(self, action, args):
         action.run_cmd(
             "docker",
             "compose",
+            "-f",
+            get_docker_compose_path(),
             "down",
-        )
+            )
 
 
 class TestGenSetupDatabaseStep(Step):
     label = "Initializing the platform database"
+    required = False
```

Comment:
@FernandezAstor , why are we changing this to `False`?

Reply:
My fault, I was using it to test it

Reply:
It was part of the oldest PR:
https://github.com/DataKitchen/data-observability-installer/pull/35/files

Reply:
Oh, I see.. Wait.. you were adding the timeout to the network removal, right? What happened to that?

Reply:
yes, but Aarthy told me to leave it as it was originally.